tracks/rust/exercises/clock/mentoring.md

```diff
@@ -40,3 +47,13 @@ impl fmt::Display for Clock {
     }
 }
 ```
+
+### Notes
+
+If somebody visibly laments the behavior of `%` on negative arguments, you can tell them how to use unstable features like `rem_euclid`.
+
+"Clean logic" for the modulus ideally means that there is usage of a function that returns a positive modulus.  (basically an implementation of `rem_floor`, or usage of the unstable `rem_euclid`).  Notice that, if the clock stores hours in addition to minutes, the corresponding `div_floor` or `div_euclid` are also necessary.
```

Comment:
I feel that a code example here would be best, instead of elaborating this "verbally". Also, providing a sample response to give to the student, in this case, would be especially helpful.